### PR TITLE
Updated the kubectl install guide

### PR DESCRIPTION
**Pull request by dipankardas011**
Signed-off-by: Dipankar Das <dipsonu10@hotmail.com>

modified:   content/en/docs/tasks/tools/install-kubectl-linux.md
modified:   content/en/docs/tasks/tools/install-kubectl-macos.md
modified:   content/en/docs/tasks/tools/install-kubectl-windows.md

Updated the kubectl version check command for all available output format

### CODE REVIEW

File: content/en/docs/tasks/tools/install-kubectl-linux.md [hunk 1]
```diff
@@ -97,8 +97,8 @@ For example, to download version {{< param "fullversion" >}} on Linux, type:
    ```
    Or use this for detailed view of version:
 
-   ```cmd
-   kubectl version --client --output=yaml
```
Comment (sftim) [inline content/en/docs/tasks/tools/install-kubectl-linux.md:101]: In documentation, we prefer long option names (eg `--option`) over short ones (`-o`). The long names are easier to memorize.

What we could do instead: link to a page that explains how different output formats work - eg https://kubernetes.io/docs/reference/kubectl/overview/#output-options

Comment (dipankardas011) [reply content/en/docs/tasks/tools/install-kubectl-linux.md:101]: Changed as per your suggestion

**Comment by dipankardas011** [comment]
Are there any problem with this PR then I can fix it 🙂

**Comment by reylejano** [comment]
Hi @dipankardas011 ,

I actually prefer the original content and at the moment this PR will have a merge conflict so a rebase is needed

I'm closing this PR but feel free to reopen this PR with `/reopen` when you can rebase

/close

**Pull request closed without merging**